x/shield/keeper/withdraw.go

```diff
@@ -4,10 +4,17 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
 
 	"github.com/certikfoundation/shentu/x/shield/types"
 )
 
+type UnbondingInfo struct {
+	Delegator      sdk.AccAddress
+	Validator      sdk.ValAddress
+	CompletionTime time.Time
+}
```

Comment:
I don't have a strong opinion on this, but I'm reluctant to move it to types/types.go because it only contains persistent state types. I just kept this type here and made it unexportable.

Reply:
I don't have a strong opinion on this, but I'm reluctant to move it to types/types.go because it only contains persistent state types. I just kept this type here and made it unexportable. Let me know if you disagree.

Reply:
I don't have a strong opinion on this, but I'm reluctant to move it to types/types.go because it only contains persistent state types. I just kept this type here and made it unexportable. Let me know if you think otherwise.